service/src/main/kotlin/fi/espoo/evaka/reports/RawReport.kt

```diff
@@ -58,9 +58,29 @@ class RawReportController(private val accessControl: AccessControl) {
 }
 
 fun Database.Read.getRawRows(from: LocalDate, to: LocalDate): List<RawReportRow> {
+    val workingDayHours = 7.75
+    val defaultOccupancyCoefficient = 7
```

Comment:
These constants are defined also somewhere else (I've seen them). Would it be better to have them only once?